Add tooltip & faq for snapshot block info

cdt-eth opened the pull request.

cryptoseneca reviewed (commented):
Two issues:

- Let's change the copy of the tooltip to be targeting the voter:
`Voters with [community_name] NFTs in their wallets before the snapshot block are eligible to vote`

- There's an issue in FAQs with apostrophes:
<img width="561" alt="Screen Shot 2022-12-08 at 10 22 00 AM" src="https://user-images.githubusercontent.com/85328329/206484780-b8bcfecd-8bde-453f-8821-66953d5ce1df.png">
<img width="1032" alt="Screen Shot 2022-12-08 at 10 14 31 AM" src="https://user-images.githubusercontent.com/85328329/206484791-7ea93a31-ca56-47f6-b5d8-49208d439b7b.png">

cdt-eth commented:
> Two issues:
> 
> * Let's change the copy of the tooltip to be targeting the voter:
>   `Voters with [community_name] NFTs in their wallets before the snapshot block are eligible to vote`
> * There's an issue in FAQs with apostrophes:
> 
> <img alt="Screen Shot 2022-12-08 at 10 22 00 AM" width="561" src="https://user-images.githubusercontent.com/85328329/206484780-b8bcfecd-8bde-453f-8821-66953d5ce1df.png">
> 
> <img alt="Screen Shot 2022-12-08 at 10 14 31 AM" width="1032" src="https://user-images.githubusercontent.com/85328329/206484791-7ea93a31-ca56-47f6-b5d8-49208d439b7b.png">

Updated Tooltip -> https://github.com/Prop-House/prop-house-monorepo/pull/343/commits/dfa1dd3fab160a3d1ddbb2c82db21d3c6c763d9a

Fixed `&rsquo;` -> https://github.com/Prop-House/prop-house-monorepo/pull/343/commits/8e1cb4fd1e5d898061cad3d55029fbbc536f0906
